deploy/examples/filesystem-ec.yaml

```diff
@@ -1,3 +1,15 @@
+# create default csi subvolume group
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystemSubVolumeGroup
+metadata:
+  name: myfs-csi # lets keep the svg crd name same as `filesystem name + csi` for the default csi svg
+  namespace: rook-ceph # namespace:cluster
+spec:
+  # The name of the subvolume group. If not set, the default is the name of the subvolumeGroup CR.
+  name: csi
+  # filesystemName is the metadata name of the CephFilesystem CR where the subvolume group will be created
+  filesystemName: myfs
+---
 #################################################################################################################
```

Comment:
This header description in each of the example files should be before the new svg cr.

Reply:
And actually I'd suggest to put the svg cr at the end of the file. For readability, they will be interested to see the filesystem CR first.

Reply:
UPdated